contacts.js

```diff
@@ -51,11 +55,34 @@ module.exports.addContact = async function (name, email, phone) {
   };
 
   contacts.push(newContact);
+  contacts.sort(function (a, b) {
+    if (a.id > b.id) {
+      return 1;
+    }
+    if (a.id < b.id) {
+      return -1;
+    }
+    return 0;
+  });
   const newContactListAsJSON = JSON.stringify(contacts);
   fs.writeFile(contactsPath, newContactListAsJSON);
   console.table(contacts);
 };
 
-// getContactById(2);
-// removeContact(2);
-// addContact('asdasd', 'asdasdad', '2323232');
+module.exports.update = async function (contact) {
+  const contacts = await fs
+    .readFile(contactsPath, { encoding: 'utf-8' })
+    .then((data) => JSON.parse(data));
```

Comment:
remove all 'then' promises

you should use only async/await style

no not mix then with await, use only await